### ISSUE TITLE
On screen keyboard

### ISSUE THREAD

**Issue by shubham1172** [shubham1172/remote-play#23]
Sometimes one might want to enter text from their remote. Allow entering text!!

**Comment by Clickedbigfoot** [comment]
I have a solution that calls a prompt to input text. It's nice as it allows android users to use swype and this doesn't need to download/create a whole new virtual keyboard. Would that suffice or do we want an entire virtual keyboard to appear on the screen?

**Comment by shubham1172** [comment]
Yes, absolutely. For the prompt, we can customize such that the user actually does not see the characters there. It could just relay the key presses to the server. Just an idea, but feel free to suggest alternatives.

**Comment by Clickedbigfoot** [comment]
Sorry, I think I misspoke; my solution calls `window.prompt()` so there isn't much room for customization there to my knowledge

**Comment by shubham1172** [comment]
Ah, I see. So instead of relaying key presses, we first take the input from the user and then pass on the string. 

The user needs to make sure to select some active text input on their computer (using the trackpad), and then write their string in the prompt which we send over to the server and make pyautogui copy the text to the actual input.

I can see this totally working, but I am thinking about the flow between the user and the computer: Say I want to search for "The Office" on Netflix. I'd probably write "The O" and the autocomplete on Netflix will suggest "The Office" based on my history. If I type letter-by-letter, it works better most of the times and feels natural (since most use cases of this will be search).  

Anyway, this was just a thought, which by the way is totally okay to park for later. Let me know what you think.

**Comment by Clickedbigfoot** [comment]
The autocomplete is a good point - we rarely type things in their entirety before seeing the target in the suggestions. I'll take a deeper look into making this more of an instant-feedback keyboard option then

**Comment by AvijitReact** [comment]
Can I work on this issue? Is this repo still active?

**Comment by Clickedbigfoot** [comment]
Hi Avijit, feel free to work on this. I have fallen behind on time

**Comment by AvijitReact** [comment]
Thanks Brandon and Shubham. I will start working on it.

**Comment by shubham1172** [comment]
@AvijitReact feel free to use this thread for any discussions as you make progress on the task.
